cypress/integration/1-getting-started/e2e.spec.js

```diff
@@ -0,0 +1,114 @@
+describe("Open Page", () => {
+    beforeEach(() => {
+      cy.visit('./source/index.html');
+    });
+    // This test verifies the main page
+    it('loads and displays main', () => {
+      cy.get('.featured').should('be.visible');
+    });
+
+    // This test verifies the wrapper components
+    it('displays three recipe wrappers', () => {
+      cy.get('.recipes-wrapper').should('have.length', 3);
+      cy.get('.recipes-wrapper').each((element, index, list) => {
+          // expect(Cypress.$(element)).to.be.visible; // true when successfully retrieve carousels -> recipes from api
+          expect(index).to.be.greaterThan(-1);
+          expect(list).to.have.length(3);
+      });
+    });
+
+    // This test types in search bar a keyword and checks if top results are shown
+    it('searches keyword:tomato', () => {
+      cy.get('#heroSearch').type('tomato');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group11/cse110-sp21-group11/pullRequest?prid=8486342)
```suggestion
      cy.get("#heroSearch").type('tomato');
```